app/controllers/users.js

```diff
@@ -37,3 +38,17 @@ exports.signin = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.getUsers = async (req, res, next) => {
+  try {
+    const { page, limit, offset } = paginationReq(req.query);
+    const { users, count } = await getUsers(offset, limit);
+
+    const userList = users.map(user => userSerializer(user));
+    const pagination = paginationData({ total: count, page, limit });
+    res.status(200).send({ users: userList, pagination });
```

Comment:
You can create a serializer to generate the entire response with the data and pagination infomartion (current page, total, etc). What do you think?